changelog.json

```diff
@@ -1,5 +1,15 @@
 {
   "releases": {
+    "1.1.2-beta3": [
+      "[Added] Syntax highlighting for Haxe files - #4445. Thanks @Gama11!",
+      "[Added] Syntax highlighting for R files - #4455. Thanks @say25!",
+      "[Fixed] Fetch to ensure \"Open in Desktop\" has a branch to checkout - #3006",
+      "[Fixed] Handle the click event when opening a binary file - #4446",
+      "[Fixed] Skip fast-forwarding when there are a lot of eligible local branches - #4392",
+      "[Fixed] Image diffs not working for files with upper-case file extension - #4466",
```

Comment:
Do we want to change this to something like
`Allow image diffs to work for files with upper-case file extensions`

Reply:
I had a chat with @nerdneha about this, and my feels on this were to describe the problem that's been fixed here. For `New` or `Added` tasks it might make sense to phrase it differently.